apps/atomy-q/API/app/Http/Controllers/Api/V1/UserController.php

```
@@ -44,12 +81,31 @@ public function index(Request $request): JsonResponse
+
+        $email = strtolower(trim((string) $validated['email']));
+        if ($this->users->emailExists($email, null, $tenantId)) {
+            return response()->json([
+                'message' => 'A user with that email already exists.',
+            ], 409);
+        }
+
+        $created = $this->userPersist->create([
+            'tenant_id' => $tenantId,
+            'email' => $email,
+            'password_hash' => $this->passwordHasher->hash(Str::random(32)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'invite'. (undefined)

(StaticAccess)